plat/arm/common/aarch64/arm_common.c

```diff
@@ -163,6 +163,7 @@ const mmap_region_t *plat_arm_get_mmap(void)
 	return plat_arm_mmap;
 }
 
+#ifdef ARM_SYS_CNTCTL_BASE
```

Comment:
I agree, it was not intentional and is due to the integration branch and the master branch
not being in sync at the moment. Will be fixed soon.